src/models/db.js

```diff
@@ -77,6 +78,13 @@ export default function() {
       deferreds.pop().resolve(db);
     }
     state = READY;
+
+    // Load default permissions.
+    const permissions = config.get('permissions').map(permission => {
```

Comment:
I just though the config would be a good place cause it allows overriding the list of default permissions (for future testing purposes for example) and it provides an easy way to access the list (`config.get('permissions'`).

I don't have a strong about this, so we could use an independent json file if you think is more appropriate.